options/parse_configfile.c

```diff
@@ -132,7 +132,8 @@ int m_config_parse(m_config_t *config, const char *location, bstr data,
             m_profile_set_desc(profile, value);
             res = 0;
         } else {
-            res = m_config_set_profile_option(config, profile, option, value);
+            res = m_config_set_profile_option(config, profile, option, value,
+                                              flags & M_SETOPT_NO_BLACKLIST);
```

Comment:
Can you maybe put a 1-line comment on why you mask everything else from `flags` here?

Reply:
There is no need to pass the `flags` to `m_config_set_profile_option`. It's my mistake.